bouter/experiment.py

```diff
@@ -26,7 +27,14 @@ class Experiment(dict):
         behavior_log=["behavior_log"],
     )
 
-    def __init__(self, path, session_id=None):
+    def __init__(
+        self, path, session_id=None, cache_active=True, default_cached=True
+    ):
+
+        # If true forces to use cached with whatever params it was computed:
+        self.default_cached = default_cached
+        self.cache_active = cache_active
```

Comment:
what does cache_active mean?

Reply:
If false, nothing is saved and nothing is read - it disables "cache mode"